projects/clickhouse/project.yaml

```diff
@@ -4,3 +4,6 @@ primary_contact: "clickhouse-feedback@yandex-team.com"
 auto_ccs:
  - "security@yandex-team.com"
```

Comment:
I'll note that using lists for auto_ccs/primary_contact can cause frustration when list members want to access bugs they get emails for but can't.

Reply:
Thanks, hope that'll be ok b/c email will be handled by task tracker and visible only to project-specific people.